### PR TITLE
set content type for callback request

### PR DESCRIPTION
**Pull request by vaputa**
It seems that the content-type of callback is missing. So I created PR and hope it can help :-)

### CODE REVIEW

**Comment by alexellis** [comment]
Hi @vaputa missed this.. thanks for contributing. We'll take a look

**Comment by alexellis** [comment]
@vaputa to merge this change you need to amend your commit with `git commit --amend -s` and then re-push

**Comment by vaputa** [comment]
@alexellis Thanks for replying. Re-pushed.

**Comment by alexellis** [comment]
I'll review the change again. Thanks or resubmitting.

**Comment by vaputa** [comment]
@alexellis Can we merge this PR to master?

File: queue-worker/main.go [hunk 1]
```diff
@@ -176,6 +176,8 @@ func postResult(client *http.Client, req queue.Request, result []byte, statusCod
 	}
 
 	request, err := http.NewRequest("POST", req.CallbackURL.String(), reader)
+	request.Header.Set("Content-Type", req.Header.Get("Content-Type"))
```
Comment (alexellis) [inline queue-worker/main.go]: I actually wonder if this is enough. Perhaps we should be setting _all_ of the headers that came from the responding function.

**Comment by alexellis** [comment]
Before merging please could you change "POST" to `http.MethodPost` or whatever the equivalent is?

**Comment by alexellis** [comment]
Seeing a CI error `./main.go:118:52: undefined: urlFunction` and also can't be rebased due to conflicts. Can you rebase with master please?

**Comment by vaputa** [comment]
@alexellis  Sorry about this. Rebase now.

**Comment by alexellis** [comment]
Merged, thank you @vaputa - for a follow-up PR we need to copy all the response headers not just the content-type.

**Comment by vaputa** [comment]
How can I help?

**Comment by alexellis** [comment]
The comment was to please copy all the response headers not just content-type. This may be done now, not sure - check it out. @vaputa

**Pull request merged**